src/pointfree/last.js

```diff
@@ -0,0 +1,43 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
+
+const isArray = require('../core/isArray')
+const isFunction = require('../core/isFunction')
+const isIterable = require('../core/isIterable')
+const isString = require('../core/isString')
+const cloneIterable = require('../core/cloneIterable')
```

Comment:
I get this is copied from head, but please move clone to the top of the list require/imports are done in alphabetical order. Feel free to change head too if you like :)